java/ql/src/experimental/Security/CWE/CWE-036/OpenStream.ql

```diff
@@ -33,11 +41,7 @@ class RemoteURLToOpenStreamFlowConfig extends TaintTracking::Configuration {
 
   override predicate isSource(DataFlow::Node source) { source instanceof RemoteFlowSource }
 
-  override predicate isSink(DataFlow::Node sink) {
-    exists(MethodAccess m |
-      sink.asExpr() = m.getQualifier() and m.getMethod() instanceof URLOpenStreamMethod
-    )
-  }
+  override predicate isSink(DataFlow::Node sink) { sinkNode(sink, "url-open-stream") }
```

Comment:
We shouldn't extend the csv models in queries, as all the csv data should have bi-directional imports to `ExternalFlow.qll`. So let's just leave the current sink as-is, and only add the new ones via csv.
```suggestion
  override predicate isSink(DataFlow::Node sink) {
    exists(MethodAccess m |
      sink.asExpr() = m.getQualifier() and m.getMethod() instanceof URLOpenStreamMethod
    )
    or
    sinkNode(sink, "url-open-stream")
  }
```